pkgs/development/interpreters/python/wrapper.nix

```diff
@@ -23,17 +23,29 @@ let
       fi
       mkdir -p "$out/bin"
 
-      for path in ${stdenv.lib.concatStringsSep " " paths}; do
-        if [ -d "$path/bin" ]; then
-          cd "$path/bin"
-          for prg in *; do
-            if [ -f "$prg" ]; then
-              rm -f "$out/bin/$prg"
-              makeWrapper "$path/bin/$prg" "$out/bin/$prg" --set PYTHONHOME "$out"
-            fi
+      # Generate new wrapper scripts for the all the executables found in our
```

Comment:
Could this change conceivably be applied to Nixpkgs without having the change to `makeWrapper` first?